package.json

```diff
@@ -102,6 +102,8 @@
     "@types/node": "^20",
     "@types/react": "^18.0.1",
     "@types/react-dom": "^18",
+    "@types/react-gtm-module": "^2.0.3",
+    "@types/react-scrollspy": "^3.3.9",
```

Comment:
You would have to move these two to `dependencies` instead of `devDependencies` since the `production` flag set on netlify doesn't allow devDeps to be installed which produces an error while compiling.